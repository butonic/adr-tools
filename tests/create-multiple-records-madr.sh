adr init docs madr
adr new First Decision
adr new Second Decision
adr new Third Decision
ls docs
head -8 docs/0003-second-decision.md
head -8 docs/0004-third-decision.md
