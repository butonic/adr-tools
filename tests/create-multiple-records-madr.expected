adr init docs madr
docs/0001-record-architecture-decisions.md
adr new First Decision
docs/0002-first-decision.md
adr new Second Decision
docs/0003-second-decision.md
adr new Third Decision
docs/0004-third-decision.md
ls docs
0001-record-architecture-decisions.md
0002-first-decision.md
0003-second-decision.md
0004-third-decision.md
head -8 docs/0003-second-decision.md
# Second Decision

* Status: Accepted
* Deciders: [list everyone involved in the decision] <!-- optional -->
* Date: 1992-01-12

Technical Story: [description | ticket/issue URL] <!-- optional -->

head -8 docs/0004-third-decision.md
# Third Decision

* Status: Accepted
* Deciders: [list everyone involved in the decision] <!-- optional -->
* Date: 1992-01-12

Technical Story: [description | ticket/issue URL] <!-- optional -->

